examples/chip-tool/commands/common/ModelCommand.cpp

```diff
@@ -81,6 +81,13 @@ void ModelCommand::SendCommand(ChipDeviceController * dc)
     dc->SendMessage(NULL, buffer);
 }
 
+bool _isValidFrame(uint8_t frameControl)
+{
+#define kZCLGlobalCmdFrameControlHeader 8
```

Comment:
These are going to be scoped outside the function anyway, so better to put them outside.